.licenserc.yaml

```diff
@@ -21,17 +21,62 @@ header:
   license:
     spdx-id: Apache-2.0
     copyright-owner: Apache Software Foundation
+    content: | # `license` will be used as the content when `fix` command needs to insert a license header.
+      Licensed to Apache Software Foundation (ASF) under one or more contributor
+      license agreements. See the NOTICE file distributed with
+      this work for additional information regarding copyright
+      ownership. Apache Software Foundation (ASF) licenses this file to you under
+      the Apache License, Version 2.0 (the "License"); you may
+      not use this file except in compliance with the License.
+      You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+      Unless required by applicable law or agreed to in writing,
+      software distributed under the License is distributed on an
+      "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+      KIND, either express or implied.  See the License for the
+      specific language governing permissions and limitations
+      under the License.
+
+    # `pattern` is optional regexp if all the file headers are the same as `license` or the license of `spdx-id` and `copyright-owner`.
+    pattern: |
+      Licensed to the Apache Software Foundation under one or more contributor
+      license agreements. See the NOTICE file distributed with
+      this work for additional information regarding copyright
+      ownership. The Apache Software Foundation licenses this file to you under
+      the Apache License, Version 2.0 \(the "License"\); you may
+      not use this file except in compliance with the License.
+      You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+      Unless required by applicable law or agreed to in writing,
+      software distributed under the License is distributed on an
+      "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+      KIND, either express or implied.  See the License for the
+      specific language governing permissions and limitations
+      under the License.
+
+  paths: # `paths` are the path list that will be checked (and fixed) by license-eye, default is ['**'].
+    - '**'
 
   paths-ignore:
     - '**/gen-codes/**'
     - '**/*.md'
-    - .gitignore
-    - .gitmodules
-    - .gitattributes
-    - LICENSE
-    - NOTICE
-    - go.mod
-    - go.sum
-    - plugins/queue/mmap/queue_operation.go
+    - 'dist'
+    - 'licenses'
+    - '**/go.mod'
+    - '**/go.sum'
+    - 'LICENSE'
+    - 'NOTICE'
+    - '.gitignore'
+    - '.gitmodules'
+    - '.gitattributes'
+    - 'plugins/queue/mmap/queue_operation.go'
```

Comment:
We skip this for?

Reply:
The file comes from https://github.com/grandecola/bigqueue and it's listed in LICENSE file

Reply:
@mrproliu Let's add a comment to indicate this.

Reply:
Have added.